### DISCUSSION TITLE
Startup Crash

### DISCUSSION THREAD

**Discussion by BarbarianBling** [valinet/ExplorerPatcher#1537] [Q&A]
Every time I click on the application it crashes immediately, everything goes blank and then nothing happens.

**Comment by pyrates999** [comment]
Uninstall explorer patcher: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

Then reboot

Then run winver and tell us your version of windows and the build number.

**Reply by krowvin** [reply]
## Something very similar happened to me

My file explorer (`explorer.exe`) was crashing every few seconds. I couldn't even stop it in task manager. It would start itself again.

 I tried switching to a different user and it was still doing it. 

**It was beyond frustrating.** Because the error in event viewer also did not lead me to think it was explorerpatcher.  

```
Faulting application name: explorer.exe, version: 10.0.22621.1635, time stamp: 0xa4159e29
Faulting module name: windowsudk.shellcommon.dll, version: 10.0.22621.1635, time stamp: 0x86ce76ab
Exception code: 0xc0000005
Fault offset: 0x0000000000262510
Faulting process id: 0x0x7A94
Faulting application start time: 0x0x1D99B334CA56696
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\SYSTEM32\windowsudk.shellcommon.dll
Report Id: d315be50-4a2a-4dae-8892-ead57231b758
Faulting package full name: 
Faulting package-relative application ID: 
```

> Uninstall explorer patcher  

As soon as I did this the issue went away.  

> version of windows and the build number   

Version 22H2 (OS Build 22621.1702)

**Comment by pyrates999** [comment]
Try the following:

1. uninstall explorer patcher
2. export this registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher" and upload it to here
3. delete the same registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
4. install explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
